pyvista/plotting/mapper.py

```diff
@@ -239,10 +229,38 @@ def configure_scalars_mode(
             n_colors,
             preference,
             interpolate_before_map,
-            rgb,
-            _custom_opac,
+            direct_scalars_color_mode,
         ):
-            """Configure scalar mode."""
+            """Configure scalar mode.
+
+            Parameters
+            ----------
+            scalars : numpy.ndarray
+                Array of scalars to assign to the mapper.
+
+            mesh : pyvista.Dataset
+                Dataset to assign the scalars to.
+
+            title : str
+                Name to assign the scalars.
+
+            n_colors : int
+                Number of colors.
+
+            preference : str
+                Either ``'point'`` or ``'cell'``.
+
+            interpolate_before_map : bool
+                Enabling makes for a smoother scalars display.  Default is
+                ``True``.  When ``False``, OpenGL will interpolate the
```

Comment:
This function doesn't have defaults, so this is a bit confusing. Then again this docstring probably doesn't get rendered anywhere.

Reply:
Documentation clarified in 129477c3c4867152eb1dfce0f50730aa61c3bc0c.